docs/resources/lke_cluster.md

```diff
@@ -193,3 +193,45 @@ In this case, the first node pool from the original configuration will be update
 the second node pool's configuration.
 
 Although not ideal, this functionality guarantees that updates to nested node pools will be reliable and predictable.
+
+## Externally Managed Node Pools
+
+By default, the `linode_lke_cluster` resource will account for all node pools under the corresponding cluster, meaning
+any node pools created externally or managed by other resources will be removed on subsequent applies.
+
+To signal the provider to ignore externally managed node pools, the `external_pool_tags` attribute can be defined with
+tags matching a tag on an externally managed node pool.
+
+For example:
+
+```terraform
+locals {
+  external_pool_tag = "external"
+}
+
+resource "linode_lke_cluster" "my-cluster" {
+    label       = "my-cluster"
+    k8s_version = "1.28"
+    region      = "us-mia"
+    
+    # This tells the Linode provider to ignore 
+    # node pools with the tag `external`, preventing
+    # externally managed node pools from being deleted.
+    external_pool_tags = [local.external_pool_tag]
+    
+    # Due to certain restrictions in Terraform and LKE, 
+    # the cluster must be defined with at least one node pool.
+    pool {
+        type  = "g6-standard-1"
+        count = 1
+    }
+}
+
+resource "linode_lke_node_pool" "my-pool" {
+  cluster_id  = linode_lke_cluster.my-cluster.id
+  type        = "g6-standard-2"
+  node_count  = 3
+
+  tags = [local.external_pool_tag]
+}
```

Comment:
I know it's a bit redundant to include this example in both files, but I figure the less clicking around readers have to do the better